src/Shared/Alert/Alert.css

```diff
@@ -0,0 +1,21 @@
+/* .snackbar-class{
+    display: block !important;
+    z-index: 98 !important;
+    color: var(--whitepeach-color) !important;
+    background-color: transparent !important;
+}
+
+.snackbar-class .alert{
+    width: 100% !important;
+} */
+
+.snackbar-class{
+    display: block;
+    z-index: 98 ;
```

Comment:
try to keep your z-index value to a minimum

Reply:
ok!